cmd/zfs_object_agent/zettaobject/src/pool_destroy.rs

```diff
@@ -116,9 +116,8 @@ impl DestroyingPoolsMap {
         destroying_pool.cache_phys.state = PoolDestroyState::Complete;
 
         debug!(
-            "update_destroy_complete: guid={} total objects={}.",
-            &guid,
-            destroying_pool.destroying_phys.unwrap().total_data_objects
+            "update_destroy_complete: guid={} destroying_phys={:?}.",
+            &guid, destroying_pool.destroying_phys
```

Comment:
```suggestion
            "update_destroy_complete: {:?} {:?}.",
            guid, destroying_pool.destroying_phys
```

I don't think the `&` is needed.
By printing the guid with `:?`, it will always be formatted the same, `PoolGuid(12345)`.
The formatting of the PoolDestroyingPhys should already include `PoolDestroyingPhys`, so `destroying_phys` doesn't add much.
Same thing elsewhere in this file.